Documentation/ApiOverview/Events/EventDispatcher/Index.rst

```diff
@@ -220,24 +231,25 @@ The PHP attribute :php:`\TYPO3\CMS\Core\Attribute\AsEventListener` supports the
 following properties (which are all optional):
 
 :php:`identifier`
-    A unique identifier must be declared which identifies the event listener,
-    and orderings can be build upon the identifier. It this property is not
-    explicitly defined, the service name is used instead.
+    A unique identifier should be declared which identifies the event listener,
+    and orderings can be build upon the identifier. If this property is not
+    explicitly defined, the service name is automatically used instead.
 
 :php:`before`
     This property allows a custom sorting of registered listeners. The listener
-    is then dispatched before the given listener. The value is the identifier of
+    is then dispatched before the given listener. The values are the identifier(s) of
```

Comment:
This may be confusing to the reader: Does an event listener have multiple identifiers? That's how the last sentence can be read.

Reply:
This came due to your suggestion that multiple identifiers are allowed, maybe rephrase this to:

```
is then dispatched before the given listener. The value is the identifier of another event listeners. Also multiple event listeners can be entered here, separated by a comma.
```

?

Reply:
Yup, fine with that.
Here is the changelog with such an example:
https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/10.0/Feature-88770-PSR-14BasedEventDispatcher.html#impact

Reply:
Changed with different wording clarifying this